src/quartic/pipeline/validator/cli.py

```diff
@@ -17,36 +18,61 @@ def cli():
 @cli.command()
 def validate():
     check_for_config()
-    graph = dag_utils.get_graph()
-    if dag_utils.is_valid_dag(graph):
-        qdag = qd.QuarticDag(graph)
-        if qdag.one_step_per_ds():
-            click.secho("Pipeline is valid.", fg="green")
+    try:
+        graph = dag_utils.get_graph()
+        if dag_utils.is_valid_dag(graph):
+            qdag = qd.QuarticDag(graph)
+            if qdag.one_step_per_ds():
+                click.secho("Pipeline is valid.", fg="green")
+            else:
+                click.secho("ERROR", fg="red", bold=True)
+                click.echo("The following outputs are defined in multiple steps:")
+                msteps = qdag.multi_steps_ds()
+                for ds, steps in msteps.items():
+                    click.secho("\t{}:".format(ds), bold=True)
+                    for s in steps:
+                        click.secho("\t\t{} in {}".format(s.get_name(), s.get_file()))
         else:
-            click.secho("Error", fg="red")
-            click.echo("The following outputs are defined in multiple steps:")
-            msteps = qdag.multi_steps_ds()
-            for ds, steps in msteps.items():
-                click.secho("\t{}:".format(ds), bold=True)
-                for s in steps:
-                    click.secho("\t\t{} in {}".format(s.get_name(), s.get_file()))
+            click.secho("ERROR", fg="red", bold=True)
+            click.echo("The pipeline is not a DAG. Check for cycles.")
+    #pylint: disable=W0703
+    except Exception as e:
+        click.secho(" ".join([click.style("ERROR:", fg="red", bold=True), str(e)]))
```

Comment:
A `string.format` here might be clearer?

Reply:
"{} {}".format([click.style("ERROR:", fg="red", bold=True), str(e)])

Reply:
OK.